ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixParticipantTest.java

```diff
@@ -78,6 +79,12 @@ public HelixParticipantTest() throws Exception {
     props.setProperty("clustermap.cluster.name", clusterName);
     props.setProperty("clustermap.datacenter.name", "DC0");
     props.setProperty("clustermap.dcs.zk.connect.strings", zkJson.toString(2));
+    propsDummy = new Properties(props);
```

Comment:
since this is only used in one test, should this be created here or in that one test? I prefer the latter

Reply:
good point. Moved it to specific test.